wolfcrypt/src/asn.c

```diff
@@ -17444,10 +17444,10 @@ int ParseCRL(DecodedCRL* dcrl, const byte* buff, word32 sz, void* cm)
         return ASN_PARSE_E;
     dcrl->sigIndex = len + idx;
 
-    if (ParseCRL_CertList(dcrl, buff, &idx, idx + len) < 0)
+    if (ParseCRL_CertList(dcrl, buff, &idx, sz) < 0)
```

Comment:
Should we use dcrl->sigIndex instead of "sz" here and in the next spot?

Reply:
Yes, that would be more appropriate. Thanks!